lib/httpClient.js

```diff
@@ -144,10 +144,41 @@ function HttpClient(operation, params)
 		{
 			return self.makeRequest();
 		}
-		var interval = 1000 / params.requestsPerSecond;
-		requestTimer = new timing.HighResolutionTimer(interval, self.makeRequest);
+
+		if (params.rampUp)
+		{
+			startRampUp(params.requestsPerSecond, params.minRps, params.rampUp);
+		} else {
+			requestTimer = new timing.HighResolutionTimer(1000 / params.requestsPerSecond, self.makeRequest);
+		}
 	};
 
+	function startRampUp(maxRps, minRps, rampUp) {
+		var currentRps = minRps;
+		var reqInterval = 1000 / currentRps;
+		var diff = maxRps - minRps;
+		var increase = diff / rampUp;
+		var rampInterval = 1000;
+		var rampTimer;
+
+		requestTimer = new timing.HighResolutionTimer(reqInterval, self.makeRequest);
+		setTimeout(function() {
```

Comment:
So what you're saying is.. we can reach the max rps 1 second early, and that would be acceptable?